Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Prerequisites for using dockerfile can be seen here https://github.com/zephyrtronium/robot/issues/76
+# docker build --build-arg CONFIG=yourConfig.toml -t robot .
+
+# Main image
+FROM golang:1.23
+
+# Define config as a build argument and make it available as an environment variable
+ARG CONFIG
+ENV CONFIG=${CONFIG}
+
+# Set the Working Directory inside the container
+WORKDIR /robot
+
+# Copy the source from the current directory to the Working Directory inside the container
+COPY . .
+
+# Build Robot
+RUN go install github.com/zephyrtronium/robot
+
+# Start up robot
+ENTRYPOINT ["sh", "-c", "robot -config ${CONFIG}"]
```

Comment:
Is there a particular reason to do this instead of `ENTRYPOINT ["robot", "-config=$CONFIG}"]`?

Reply:
Think I can change it, I was mostly adapting a template I found online to suite my needs.

In troubleshooting I ran into some issues and didn't try to undo things after I reached the right result. Will clean this up tomorrow